config/settings/base.py

```diff
@@ -655,3 +672,6 @@
 TASK_SUMMARIZE_DISTRICT_PATIENT = env.bool(
     "TASK_SUMMARIZE_DISTRICT_PATIENT", default=True
 )
+
+# Time logging middleware
+ENABLE_REQUEST_TIME_LOGGING = env.bool("ENABLE_REQUEST_TIME_LOGGING", default=True)
```

Comment:
instead of setting this flag,
add the middleware to the list if the environment variable is true